nengo_viz/components/netgraph.py

```diff
@@ -27,15 +27,21 @@ def __init__(self, viz, config, uid):
         self.parents = {}
         self.networks_to_search = [self.viz.model]
         self.initialized_pan_and_zoom = False
-        self.last_modify_time = os.path.getmtime(self.viz.viz.filename)
+        try:
+            self.last_modify_time = os.path.getmtime(self.viz.viz.filename)
+        except:
+            self.last_modify_time = None;
         self.last_reload_check = time.time()
 
     def check_for_reload(self):
-        t = os.path.getmtime(self.viz.viz.filename)
+        try:
+            t = os.path.getmtime(self.viz.viz.filename)
 
-        if self.last_modify_time < t:
-            self.reload()
-            self.last_modify_time = t
+            if self.last_modify_time < t:
+                self.reload()
+                self.last_modify_time = t
+        except:
+            pass
```

Comment:
I'd much prefer if this didn't fail silently like this.  Is this meant to dealing with the filename not existing, or any possible error in the reload process?